src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-weight: 450;
+  font-family: Roboto, sans-serif;
+  font-size: 15px;
+  line-height: 14.06px;
+  background-color: #e5e5e5;
   margin: 0;
+  padding: 0;
+}
+
+header {
+  width: 100%;
+  justify-content: space-between;
+  height: 60px;
+  background-color: #fff;
```

Comment:
Consider using a CSS variable for the blue color (#00acdc) as required by the task. This will make it easier to maintain and update the color scheme.